### ISSUE TITLE
Use new *Options() over DesiredCapabilities

### ISSUE THREAD

**Issue by vmassol** [testcontainers/testcontainers-java#931]
When executing TC I get:

```
Oct 22, 2018 3:07:08 PM org.openqa.selenium.remote.DesiredCapabilities firefox
INFO: Using `new FirefoxOptions()` is preferred to `DesiredCapabilities.firefox()`
Oct 22, 2018 3:07:08 PM org.openqa.selenium.remote.DesiredCapabilities chrome
INFO: Using `new ChromeOptions()` is preferred to `DesiredCapabilities.chrome()`
```

That's because I need to pass a DesiredCapabilities to TC in:

```
        BrowserWebDriverContainer webDriverContainer = new BrowserWebDriverContainer<>()
            .withDesiredCapabilities(uiTestAnnotation.browser().getResolvedBrowser().getCapabilities())
```

It seems that DesiredCapabilitiesis now deprecated and maybe TC should have a `.withCapability(Capability capability)` method (since FirefoxOptions implements Capabilities for ex).

**Comment by rnorth** [comment]
That's a fair point, and I think we could go with that. If you or anyone else would like to raise a PR that'd be helpful.

One thing I'd like to be careful of, though, is backwards compatibility. Since we let people use quite old versions of the Selenium libs, we just need to make sure that no runtime breakages occur.

**Comment by vmassol** [comment]
> One thing I'd like to be careful of, though, is backwards compatibility. Since we let people use quite old versions of the Selenium libs, we just need to make sure that no runtime breakages occur.

I guess that could be handled by introducing a new `.withCapabilities(Capabilities)` and keeping the `.withDesiredCapabilities(DesiredCapabilities)` one (but deprecating it), right?

**Comment by kiview** [comment]
Fixed in #995

**Comment by vmassol** [comment]
Thanks and thanks to @pavelpp :)
